CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 
 ### Changed
 
+* nostr: changed args for Gift Wrapping using `EventBuilder::gift_wrap` and `EventBuilder::gift_wrap_from_seal`: removed `expiration` in favor of `extra_tags` ([erskinagrdner])
```

Comment:
Given this is a breaking change, is there anything else you'd like me to add here or anywhere else?

Reply:
Yeah, could be a good idea to document all breaking changes in a dedicated section. I'll change this in the cherry-picking